### PR TITLE
SashaSublime package

### PR DESCRIPTION
**Pull request by Kristinita**

### CODE REVIEW

File: repository/s.json [hunk 1]
```diff
@@ -86,6 +86,22 @@
 			]
 		},
 		{
+			"name": "Sasha",
+			"details": "https://github.com/Kristinita/SashaSublime",
+			"labels": ["language syntax"],
+			"homepage": "https://github.com/Kristinita/SashaSublime",
```
Comment (wbond) [inline repository/s.json]: This will be automatically selected based on `details`, so you may omit it.

File: repository/s.json [hunk 2]
```diff
@@ -86,6 +86,22 @@
...
+			"author": "Sasha Chernykh",
+			"readme": "https://github.com/Kristinita/SashaSublime/blob/master/README.md",
+			"issues": "https://github.com/Kristinita/SashaSublime/issues",
```
Comment (wbond) [inline repository/s.json]: `readme` and `issues` are also set to their default values based on `details`, so you may omit them.

**Comment by wbond** [comment]
You'll need to create a [semver](http://semver.org) tag on your repo that begins with `st3-` (since you specified that in your PR).

**Comment by Kristinita** [comment]
> This will be automatically selected based on details, so you may omit it.
> readme and issues are also set to their default values based on details, so you may omit them.

Yes, I understand and I will remove the extra lines. I need to create new pull-request?

> You'll need to create a semver tag on your repo that begins with st3- (since you specified that in your PR).

I [create](https://github.com/Kristinita/SashaSublime/releases/tag/st3-1.0.0) tag.

Thanks.

**Comment by wbond** [comment]
> Yes, I understand and I will remove the extra lines. I need to create new pull-request?

No, just create a new commit and push the comment to branch you use for this pull request.

**Pull request merged**